python/ray/autoscaler/_private/util.py

```diff
@@ -257,27 +257,23 @@ def merge_setup_commands(config):
     return config
 
 
-def fill_node_type_max_workers(config):
+def fill_node_type_min_max_workers(config):
     """Sets default per-node max workers to global max_workers.
     This equivalent to setting the default per-node max workers to infinity,
     with the only upper constraint coming from the global max_workers.
+    Sets default per-node max workers to zero.
```

Comment:
```suggestion
    Sets default per-node max workers to global max_workers.
```

Reply:
We have. comment about max workers above.
Changes this one to talk about min workers.

Reply:
my main concern was that it says "Sets default per-node max workers to zero." but it actually sets it to global max workers. Maybe you meant min_workers to zero?